dotnet/src/SemanticKernel.Abstractions/Diagnostics/MicrosoftDiagnostics.cs

```diff
@@ -0,0 +1,46 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+
+namespace Microsoft.SemanticKernel.Diagnostics;
+
+public static class MicrosoftDiagnostics
```

Comment:
I feel like this class name should have something more focused, since the namespace is already Diagnostics, and the only thing Microsoft-specific is that Azure env key. Can we use something with "Telemetry"?

Reply:
tbh I don't care about the name, I'll change it to Telemetry :-) 

My main concern is that someone will reuse this code for other things and we might end up breaking the integration with Azure Core SDK